src/databricks/labs/ucx/framework/crawlers.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
```

Comment:
Why is this required?

Reply:
It lets a class use itself in type annotations, without resorting to strings.

Reply:
ah, thanks, did not know that